src/cpp/multisolver_interface/environment.cc

```diff
@@ -207,7 +207,7 @@ void printXpressBanner(bool error) {
   XPRSgetbanner(banner);
 
   if (error) {
-    std::cerr << "Xpress banner :\n" << banner << "\n";
+    std::cout << "Xpress banner :\n" << banner << "\n";
```

Comment:
What is the purpose of the if block now ?

Reply:
To print xpress  beginning message

Reply:
Yes but the `if` and `else` have the same code, either remove the `if/ellse` structure or correct the code as needed